src/almanack/code_tracker.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module calculates the absolute value of lines of code (LoC) changed (added or removed)
+between the source and target commit hash.
+"""
+
+import pathlib
+
+from .git_parser import get_commit_logs
+
+
+def calculate_loc_changes(repo_path: pathlib.Path, source: str, target: str) -> int:
```

Comment:
Would it make sense to include this function within the `git_parser` module? I ask as it seems to use similar parameters / attributes. If you opt to include it in the same module, it could help with the organization of the package and development.